### ISSUE TITLE
Trying to get Isotope masonry working on my site

### ISSUE THREAD

**Issue by roybman** [metafizzy/isotope#1027]
I'm trying to incorporate Isotope into a site I'm coding and not able to get it to work the way I expected. The goal is to create columns of content boxes that will have a relative width (for now it's 28%) and a variable height, depending on the amount of content in the box. I want the boxes aligned into columns and have each box directly under the one above, masonry style. Testing my code on the site being built yielded no result, as can be seen here: http://bruggemann.us/ So I copied my code into codepen to try and work it out, and got a different result, albeit still not the result I'm looking for. That is at http://codepen.io/anon/pen/adGJmY For reference, I'm using in my code:

```
<script src="web/js/jquery-1.12.0.min.js"></script>
<script src="web/js/material.min.js"></script>
<script src="web/js/isotope.pkgd.min.js"></script>

```
I've looked at the documentation and looked at other codepen samples, and can't find an answer to what I may be missing. Any help is greatly appreciated.

**Comment by desandro** [comment]
Try including `web/js/imagesloaded.pkgd.min.js` before your `$(document).ready` code.

**Comment by roybman** [comment]
Long story short, in trying to figure out my issue, I boiled the code down to it's simplest so I could try to figure out what was wrong, I mistakenly commented out the isotope script file. I've now realized the original issue was I didn't include my own js file in a script tag, so of course, it won't work. I've put my original code back together and ensured I have a pointer to my js file, and it works. I apologize for wasting your time.  It was my own stupid mistake, and as sometimes happens, it is one of the most obvious things that get overlooked.
